tests/e2e/vsphere_volume_expansion.go

```diff
@@ -1983,11 +2072,20 @@ func waitForFSResize(pvc *v1.PersistentVolumeClaim, c clientset.Interface) (*v1.
 }
 
 // getFSSizeMb returns filesystem size in Mb
-func getFSSizeMb(f *framework.Framework, pod *v1.Pod) (int64, error) {
-	output, err := storage_utils.PodExec(f, pod, "df -T -m | grep /mnt/volume1")
-	if err != nil {
-		return -1, fmt.Errorf("unable to find mount path via `df -T`: %v", err)
+func getFSSizeMb(f *framework.Framework, pod *v1.Pod, namespace string) (int64, error) {
```

Comment:
instead of changing this everywhere, I see two options,
1. use variadic argument for namespace
2. for SVC case we can use `getNamespaceToRunTests()`

Reply:
Used getNamespaceToRunTests()  in the code. Reverted the previous changes